### DISCUSSION TITLE
Support for Synchronized Charts

### DISCUSSION THREAD

**Discussion by yesterdayforgotten** [RomRider/apexcharts-card#513] [Ideas]
ApexCharts.js supports having multiple charts which support a common and synchronized x-axis:
[https://apexcharts.com/docs/chart-types/synchronized-charts/](https://apexcharts.com/docs/chart-types/synchronized-charts/)
![image](https://user-images.githubusercontent.com/5847795/222837951-95d0bb82-78d7-4be7-bf18-d061df0d4dc0.png)

Would it be possible to support this in this card as well?

It would be useful for me to have separate graphs for certain data with completely independent y-axes and to be able to display the data in a stacked fashion so that the graphs don't overlap, but still be able to synchronize the viewing of the values at any given point. 

Today I can create multiple instances of the card and stack them in a vertical stack card, but the result is not very compact and there is no synchronization between them.

**Comment by heinemannj** [comment]
I've created a feature request: https://github.com/RomRider/apexcharts-card/issues/703
